### ISSUE TITLE
Menu not work proper

### ISSUE THREAD

**Issue by ghost** [Chemaclass/knob-base#7]
Hello again,

after last update the WordPress menu seem not work proper.

See this screenshot and link on the page/post not work,

If debug is set to true show this error.

`Notice: Trying to get property of non-object in C:\wamp64\www\wordpress1\wp-content\themes\knob-mvc-master\app\libs\WalkerNavMenu.php on line 81`

![menu](https://cloud.githubusercontent.com/assets/14064365/22904663/8384a30a-f23d-11e6-97fb-6676bc3e13e3.PNG)

**Comment by Chemaclass** [comment]
Right. So far the current implementation doesnt work properly with more than one sublevel on the menu... I need to fix that during the next days. But if you have any idea feel free to make a PR, for sure.
The file which this is handle is WalkerNavMenu.
